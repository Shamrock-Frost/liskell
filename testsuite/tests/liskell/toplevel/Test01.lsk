(defmodule Main _ (Prelude))

(define main undefined)
