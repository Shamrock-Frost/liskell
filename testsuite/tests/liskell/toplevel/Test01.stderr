[1 of 1] Compiling Main             ( Test01.lsk, Test01.o )
Linking Test01 ...
